HPXMLtoOpenStudio/resources/airflow.rb

```diff
@@ -190,59 +189,33 @@ def self.apply_infiltration_to_unconditioned_space(model, space, ach = nil, ela
     end
   end
 
-  def self.apply_natural_ventilation_and_whole_house_fan(model, weather, vent_whf, open_window_area, nv_clg_ssn_sensor)
+  def self.apply_natural_ventilation_and_whole_house_fan(model, weather, vent_fans_whf, open_window_area, nv_clg_ssn_sensor)
     if @living_zone.thermostatSetpointDualSetpoint.is_initialized
       thermostat = @living_zone.thermostatSetpointDualSetpoint.get
       htg_sch = thermostat.heatingSetpointTemperatureSchedule.get
       clg_sch = thermostat.coolingSetpointTemperatureSchedule.get
     end
 
+    # NV Availability Schedule
     nv_num_days_per_week = 7 # FUTURE: Expose via HPXML?
-    if vent_whf.nil?
-      whf_num_days_per_week = 0
-      whf_cfm = 0.0
-      whf_fan_w = 0.0
-    else
+    nv_aval_sch = create_nv_and_whf_avail_sch(model, Constants.ObjectNameNaturalVentilation, nv_num_days_per_week)
+
+    nv_avail_sensor = OpenStudio::Model::EnergyManagementSystemSensor.new(model, 'Schedule Value')
+    nv_avail_sensor.setName("#{Constants.ObjectNameNaturalVentilation} avail s")
+    nv_avail_sensor.setKeyName(nv_aval_sch.name.to_s)
+
+    # Availability Schedules paired with vent fan class
+    # If whf_num_days_per_week is exposed, can handle multiple fans with different days of operation
+    whf_avail_sensors = {}
+    vent_fans_whf.each do |vent_whf|
       whf_num_days_per_week = 7 # FUTURE: Expose via HPXML?
-      whf_cfm = vent_whf.rated_flow_rate
-      whf_fan_w = vent_whf.fan_power
-    end
+      obj_name = "#{Constants.ObjectNameWholeHouseFan} #{vent_whf.id}"
```

Comment:
I'm a little concerned that these names could become really long (or have invalid characters?) since we're not in control of them, as they come from the user's HPXML file. How about if we just use index values (1, 2, 3) instead of ids?

Reply:
Good catch, adding id into the name is good for debugging as you can see if things in idf are consistent with HPXML, but yeah it might introduce issues.